src/main/java/com/programmers/engine/model/stack/CalculationStack.java

```diff
@@ -0,0 +1,67 @@
+package com.programmers.engine.model.stack;
+
+import com.programmers.engine.model.confirmation.Confirmation;
+import com.programmers.engine.model.operation.Operator;
+
+import java.util.List;
+import java.util.Stack;
+
+public class CalculationStack {
+
+    private final Stack<Integer> operandStack;
+    private final Stack<String> operatorStack;
```

Comment:
```suggestion
    private final Stack<Opertator> operatorStack;
```
요렇게 입력받아놓으시면 관리하기 더 좋으실거에요.